sycl/ReleaseNotes.md

```diff
@@ -1,3 +1,369 @@
+extension proposal. [f5fb7599]
+- Added the [`sycl_ext_intel_cslice`](doc/extensions/proposed/sycl_ext_intel_cslice.asciidoc)
+extension proposal. [5777e1fe]
+- Added the [`sycl_ext_oneapi_group_sort`](doc/extensions/proposed/sycl_ext_oneapi_group_sort.asciidoc)
+extension update proposal that introduced sorting functions with fixed-size arrays. [c6d1cafd]
+- Added device ID, memory clock rate and bus width device information queries to the
+[`sycl_ext_intel_device_info`](doc/extensions/supported/sycl_ext_intel_device_info.md)
+extension. [1d993446][4f7787c8]
+
+## Improvements
+
+### SYCL Compiler
+- Added the `InferAddressSpaces` pass to the SPIR/SPIR-V compilation pipeline,
+reducing the size of the generated device code. [a3ae0dd1]
+- Redesigned pointer handling so that it no longer decomposes kernel argument
+types containing pointers. [3916d3b6] [d55e9c28] [9b025068]
+- Kernel lambda operator is now always inlined in the device code entry point
+unless `-O0` is used. [b91b7327] [2359d948]
+- Improved entry point handling in the `sycl-post-link` tool. [53d9c7b6]
+- The `reqd_work_group_size` attribute now works with 1, 2 or 3 operands.
+[4ff42c3c]
+- Enabled using `-fcf-protection` option with `-fsycl`, which results in it
+being applied only to host code compilation and producing a warning. [b6f61f62]
+- Linux based compiler driver on Windows now pulls in the `sycld` debug library
+when `msvcrtd` is specified as a dependent library. [ebf6c597]
+- Added `/Zc:__cplusplus` as a default option during host compilation with MSVC.
+[e7ed8607]
+- Improved the `ESIMDOptimizeVecArgCallConv` optimization pass to cover more IR
+patterns. [49264549]
+- Added support for more types in ESIMD lsc functions. [d9e40ecd]
+- Added error diagnostics for using
+`sycl::ext::oneapi::experimental::annotated_arg/ptr` as a nested type.
+[321c733c]
+- The status of `bfloat16` support was changed from experimental to supported.
+[7b47ebb6]
+
+### SYCL Library
+- Updated `online_compiler` with Gen12 GPU support. [adfb1c10]
+- `get_kernel_bundle` and `has_kernel_bundle` now check that the kernels are
+compatible with the devices. [91b15156]
+- Waiting for an event associated with a kernel that uses a stream now also
+waits for the stream to be flushed. [1db0e81f]
+- Added the requested device type to the message of the exception thrown when no
+such devices are found. [6b83ad7c]
+- Optimized `operator[]` of `host_accessor`. [01e60f76]
+- Improved reduction performance on discrete GPUs. [99bdc823]
+- Added `invoke_simd` support for functions with `void` return type. [3fd08509]
+- The Level Zero plugin now creates every event as host-visible by default.
+[f3d245d6]
+- Added Level Zero plugin support for global work sizes greater than
+`UINT32_MAX` as long as they are divisible by some legal work-group size and
+the resulting quotient does not exceed `UINT32_MAX`. [62dd13dd]
+- Multiple CCS are now reported as sub-devices with Level Zero on single tile
+GPUs. [9cf74516]
```

Comment:
Not needed, I think, because consumed by 

> - Implemented the [`sycl_ext_intel_cslice`](doc/extensions/supported/sycl_ext_intel_cslice.asciidoc) extension. [5995c618]
> - Implemented the [`sycl_ext_intel_queue_index`](doc/extensions/proposed/sycl_ext_intel_queue_index.asciidoc) extension. [d2ec9645] [7179e830]

+ @smaslov-intel to confirm/reject.


```suggestion
```

Reply:
I am fine with that, but would like to suggest that we spell explicitly that partition of sub-devices to sub-sub-devices is a deprecated feature.

<html>
<body>
<!--StartFragment-->

SYCL_PI_LEVEL_ZERO_EXPOSE_CSLICE_IN_AFFINITY_PARTITIONING (Deprecated) | Integer | When set to non-zero value exposes compute slices as sub-sub-devices in sycl::info::partition_property::partition_by_affinity_domain partitioning scheme. Default is zero meaning that they are only exposed when partitioning by sycl::info::partition_property::ext_intel_partition_by_cslice. This option is introduced for compatibility reasons and is immediately deprecated. New code must not rely on this behavior. Also note that even if sub-sub-device was created using partition_by_affinity_domain it would still be reported as created via partitioning by compute slices.
-- | -- | --


<!--EndFragment-->
</body>
</html>